### ISSUE TITLE
trouble with auditwheel repair step since october 14

### ISSUE THREAD

**Issue by hredestig** [pypa/manylinux#80]
I have the manylinux1_x86_64 image to create python wheels and using this docker file
```
FROM quay.io/pypa/manylinux1_x86_64

ENV GLPK_VER="4.60"
RUN wget http://ftp.gnu.org/gnu/glpk/glpk-${GLPK_VER}.tar.gz -O - | tar xz
WORKDIR glpk-${GLPK_VER}
RUN ./configure && make install
```

and last week I could with this test.sh

```
#!/bin/bash
auditwheel show /io/wheelhouse/cobra-0.5.1b1-cp35-cp35m-linux_x86_64.whl
```
run 
```
docker run --rm -v `pwd`:/io cobrapy_builder /io/test.sh
```
to get expected output
```
cobra-0.5.1b1-cp35-cp35m-linux_x86_64.whl is consistent with the
following platform tag: "linux_x86_64".

The wheel references the following external versioned symbols in
system-provided shared libraries: GLIBC_2.4.

The following external shared libraries are required by the wheel:
{
    "libc.so.6": "/lib64/libc-2.5.so",
    "libglpk.so.40": "/usr/local/lib/libglpk.so.40.1.0",
    "libm.so.6": "/lib64/libm-2.5.so",
    "libpthread.so.0": "/lib64/libpthread-2.5.so"
}

In order to achieve the tag platform tag "manylinux1_x86_64" the
following shared library dependencies will need to be eliminated:

libglpk.so.40
```
now however, I only get

```
nothing to do

cobra-0.5.1b1-cp35-cp35m-linux_x86_64.whl is consistent with the
following platform tag: "manylinux1_x86_64".

The wheel references no external versioned symbols from system-
provided shared libraries.

The wheel requires no external shared libraries! :)
```

I can circumvent that by installing pyelftools==0.23 (replacing the 0.24 version) ~~but that only fixes the problem for python 3.5 not 3.4 or 2.7~~ (can't reproduce that, seems to fix for all python's), i.e.

```
#!/bin/bash
/opt/python/cp35-cp35m/bin/pip install pyelftools==0.23
auditwheel show /io/wheelhouse/cobra-0.5.1b1-cp35-cp35m-linux_x86_64.whl
```

any ideas? the wheel file I try with is here 
https://drive.google.com/open?id=0B9Jk-Vpwjhb0ME1HNEx0OFNNbVU

**Comment by hredestig** [comment]
@matthew-brett I noticed this with multibuild, did you also experience pyelftools not being able to spot dependencies in shared object files?

**Comment by sbidoul** [comment]
I was hit by this issue this morning using the quay.io/pypa/manylinux1_x86_64 docker image.

**Comment by njsmith** [comment]
I think this should have been fixed by #81 (merged this morning). Could you run `docker pull quay.io/pypa/manylinux1_x86_64` to make sure you have the latest image and try again?

**Comment by hredestig** [comment]
confirmed, works again, thanks!
